src/AppCommon/Infra/SerializationOptions.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Particular.EndpointThroughputCounter.Infra
+{
+    using System.Text.Encodings.Web;
+    using System.Text.Json;
+
+    public static class SerializationOptions
+    {
+        public static readonly JsonSerializerOptions IndentedWithNoEscaping = new()
+        {
+            WriteIndented = true,
+            //DefaultIgnoreCondition = JsonIgnoreCondition.when JsonIgnoreCondition.WhenWritingNull,
```

Comment:
Why are these commented here, they were in the other PR?

Reply:
sorted